src/components/styles/raceCreate.css

```diff
@@ -117,144 +103,101 @@ align-items: center;
     margin-right: 10px;
     margin-top: 10px;
 }
-.race-modal-container {
-    border: 1px black solid;
-    display: flex;
-    flex-direction: column;
-    height: 530px;
-    background: #F0F0F0;
+.race-modal-content-container {
+    height: fit-content
 }
-.race-modal-header-container {
-    display: flex;
-    height: 45px;
-    width: 440px;
-    margin-Left: 10px;
-    margin-Top: 10px;
-}
-.race-modal-header-confirm {
+.race-modal-accordion-link-container {
+    margin: 5px;
     display: flex;
-    font-size: 33px;
-    font-Weight: 600;
-    justify-Content: flex-start;
     align-items: center;
-    width: 200px;
-    margin-left: 10px;
+    flex-direction: column
 }
-.race-modal-header-x {
-    display: flex;
-    cursor: pointer;
-    justify-content: flex-end;
-    align-items: center;
-    font-size: 15px;
-    font-weight: bold;
-    height: 45px;
-    width: 205px;
-    margin-Left: 10px;
-    margin-right: 30px;
-}
-.race-modal-body-container {
-    height: 390px;
-    width: 440px;
-    margin-left: 10px;
-    margin-top: 10px;
-    overflow: scroll;
-    background: #454545;
+.race-modal-link-container {
+    margin: 5px;
     color: white;
-    overflow-x: hidden;
-}
-.race-modal-body-flexrow {
     display: flex;
-    height: 210px;
-    width: 420px;
-}
-.race-modal-body-flexcolumn {
-    display: flex;
-    flex-direction: column;
-    height: 190px;
-    width: 300px;
-    margin-top: 10px;
-}
-.race-modal-name {
-    font-size: 33px;
-    font-weight: bold;
-    height: 50px;
-    width: 275px;
-    margin-left: 10px;
-    margin-top: 10px;
+    justify-content: center;
+    width: 310px;
+    background: rgb(118, 30, 33);
+    opacity: .6;
+    margin-bottom: 10px;
+    cursor: pointer
 }
-.race-modal-info {
-    height: 65px;
-    width: 275px;
-    margin-left: 10px;
-    margin-top: 10px;
+.race-modal-link-container:hover {
+    opacity: .8;
+    box-shadow: 0px 1px 1px 0px black;
 }
-.race-modal-racial-trait-title {
-    display: flex;
-    align-items: center;
-    font-size: 25px;
-    font-weight: 500;
-    margin-left: 10px;
-    margin-top: 10px;
-    width: 275px;
-    height: 35px;
+.race-modal-link {
+    text-decoration: none;
+    color: white
```

Comment:
What do you think about moving colors into a colors.css file (or something similar to that) and creating variables so that you can keep the app consistent?